CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-10-10
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Draft is looking to get feedback. If you have used Draft or are interested in Draft, please click [here](https://github.com/Azure/draft/issues/140) to start a conversation with the AKS team.
```

Comment:
add a link to Draft docs for users who are new to it?

Reply:
@qpetraroia ^^

Reply:
We closed this ticket now, so we are good to take this out